src/views/About.js

```diff
@@ -0,0 +1,167 @@
+            
+               </div> 
+               <div className="card">
+              <img src={connor} alt=""/>
+               <h2>Connor Robertson</h2>
+               <p>Full Stack Developer</p>
+               <a href="https://github.com/ConConRob"><i class="fab fa-github gh"></i></a>
+               <a href="https://uk.linkedin.com/"><i class="fab fa-linkedin tw"></i></a>
+               <a href="https://twitter.com"><i class="fab fa-twitter tw "></i></a>
+               
+               </div> 
+               <div className="card">
+              <img src={mark} alt=""/>
+               <h2>Mark Marshall</h2>
+               <p>Full Stack Developer</p>
+               <a href="https://github.com/mark-marshall"><i class="fab fa-github gh"></i></a>
+               <a href="https://uk.linkedin.com/"><i class="fab fa-linkedin tw"></i></a>
+               <a href="https://twitter.com"><i class="fab fa-twitter tw "></i></a>
+              
+               </div> 
+              <div className="card">
+              <img src={pavol} alt=""/>
+               <h2>Pavol Karas</h2>
+               <p>Full Stack Developer</p>
+               <a href="https://github.com/Pav0l"><i class="fab fa-github gh"></i></a>
+               <a href="https://uk.linkedin.com/"><i class="fab fa-linkedin tw"></i></a>
+               <a href="https://twitter.com/KarasPavol"><i class="fab fa-twitter tw "></i></a>
+            
+               </div> 
+               <div className="card">
+              <img src={talent} alt=""/>
+               <h2>Talent Antonio</h2>
+               <p>Full Stack Developer</p>
+               <a href="https://github.com/sampler36"><i class="fab fa-github gh"></i></a>
+               <a href="https://uk.linkedin.com/"><i class="fab fa-linkedin tw"></i></a>
+               <a href="https://twitter.com"><i class="fab fa-twitter tw "></i></a>
+              
+               </div> 
+            </section>
+          </PageContainer>
+        </PageOuter>
+    );
+  }
+}
+
+export default About;
+
+const PageOuter = styled.div`
+background: ${theme.color.white};
+
+.header {
+  font-size:2.5rem;
+  color:${theme.color.accentPurple};
+  text-align:center;
+  padding-top: 1.5625rem;
+}
+`;
+
+const PageContainer = styled.div`
+  background: ${theme.color.lightPurple};
+  margin: 5rem 5rem;
+  box-shadow: ${theme.shadow.containerShadow};
+  border:none;
+
+  section {
+    text-align: center;
+    padding: 0.3125rem 0.125rem;
+    border:none;
+}
+  .about-container {
+      display:flex;
+      justify-content:space-evenly;
+      padding: 100px 0;
+      flex-flow: wrap;
+      
+      .card{
+        width:340px;
+        padding: 1.25rem;
+        border-bottom:0.125rem solid ${theme.color.accentPurple};
+        margin: 1rem 0.625rem;
+        &:hover {
+          background-color:${theme.color.white}
+          transition: all 0.3s ease-in;
+          transform: scale(1.2);
```

Comment:
I don't think this works for the page as it looks quite out of place. I'd recommend removing it and leaving the cards simple without any hover effects.